routing/pathfind_test.go

```diff
@@ -3500,16 +3500,19 @@ func TestBlindedRouteConstruction(t *testing.T) {
 			ToNodeFeatures: tlvFeatures,
 		}
 
-		// Create final hop parameters for payment amount = 110. Note
-		// that final cltv delta is not set because blinded paths
-		// include this final delta in their aggregate delta. A
-		// sender-set delta may be added to account for block arrival
-		// during payment, but we do not set it in this test.
+		// Create final hop parameters for payment amount = 110.
```

Comment:
Q: So we will not allow an additional block arrival padding for the blinded path case ?

Reply:
ah good point! - I think I was sort of assuming that the recipient would add this buffer to the communicated blinded path cltv delta but after looking again at the spec and the proposal doc - it looks like you are right and the sender still sets this. Will update!

Reply:
ah, although... check this reply from t-bast: https://github.com/lightning/bolts/pull/798/files#r1059339014

In it he mentions that it is potentially no longer necessary for the sender to account for this buffer since the receiver can take care of it with dummy hops (obvs then we need to make sure to actually add these dummy hops).

Reply:
that seems to be an ok approach, to let the receiver handle this, since it is already tasked with building routes that are specially selected